.rubocop.yml

```diff
@@ -281,3 +281,8 @@ Style/TrailingCommaInLiteral:
 
 Style/SingleLineBlockParams:
   Enabled: false
+
+Layout/IndentHash:
+  Enabled: true
+  Exclude:
+    - '**/*.html.slim*'
```

Comment:
Huh. This must be new. I didn't know you could disable cops within .rubocop.yml for slim-lint to ignore.